andela/mrm_api
# #164457411 An admin should be able to get structure information when querying locations

#### Description
Currently, When querying locations,  passing in children in GraphQL only brings back one level. This PR brings back all the  nodes or levels for that location
#### Type of change
- New feature (non-breaking change which adds functionality)

#### How Has This Been Tested?
- Pull this branch `story/164457411-add-structure-info-to-getLocation-query`
- Run the `allLocations` query with 
```
{
    allLocations{
        name
        abbreviation
      	structureId
        structure{
            name
            getNodes{
              id
              name
              level
            }
        }
    }
}
```
- You should be able to view all the nodes associated with that location.


#### Checklist:





#### Pivotal Tracker
[#164457411](https://www.pivotaltracker.com/story/show/164457411)
#### Screenshots if any
<img width="1025" alt="Screen Shot 2019-03-18 at 2 04 12 PM" src="https://user-images.githubusercontent.com/22786444/54526104-c5884c00-4986-11e9-99eb-7d48e2b4e29f.png">


#### Any background context that you would like to provide:
- The data returned are all the nodes associated with that location.
